constantine-go/constantine_test.go

```diff
@@ -1379,3 +1379,232 @@ func TestBatchVerify(t *testing.T) {
+	input := "38d18acb67d25c8bb9942764b62f18e17054f66a817bd4295423adf9ed98873e000000000000000000000000000000000000000000000000000000000000001b38d18acb67d25c8bb9942764b62f18e17054f66a817bd4295423adf9ed98873e789d1dd423d25f0772d2748d60f7e4b81bb14d086eba8e8e8efb6dcff8a4ae02"
+	expected := "811c7003375852fabd0d362e40e68607a12bdabae61a7d068fe5fdd1dbbf2a5d"
+	fmt.Println("Running SHA256 tests")
+
+	var inputBytes []byte
+	inputBytes = make([]byte, len(input) / 2, len(input) / 2)
+	var expectedBytes []byte
+	expectedBytes = make([]byte, len(expected) / 2, len(expected) / 2)
+
+	err := fromHexImpl(inputBytes[:], []byte(input))
+	if err != nil {
+		require.True(t, false)
+	}
+	err = fromHexImpl(expectedBytes[:], []byte(expected))
+	if err != nil {
+		require.True(t, false)
+	}
+
+	var r [32]byte
+
+	status, err := EvmSha256(&r, inputBytes)
+	if status != true {
+		require.True(t, false)
+	}
+
+	require.Equal(t, r[:], expectedBytes[:])
+	fmt.Println("Success")
+}
+
+var (
+	testDirEvm						= "../tests/protocol_ethereum_evm_precompiles/"
+
+	modexp_tests					= filepath.Join(testDirEvm, "modexp.json")
+	modexp_eip2565_tests			= filepath.Join(testDirEvm, "modexp_eip2565.json")
+
+	bn256Add_tests					= filepath.Join(testDirEvm, "bn256Add.json")
+	bn256ScalarMul_tests			= filepath.Join(testDirEvm, "bn256ScalarMul.json")
+	bn256Pairing_tests				= filepath.Join(testDirEvm, "bn256Pairing.json")
+
+	add_G1_bls_tests				= filepath.Join(testDirEvm, "eip-2537/add_G1_bls.json")
+	fail_add_G1_bls_tests			= filepath.Join(testDirEvm, "eip-2537/fail-add_G1_bls.json")
+	add_G2_bls_tests				= filepath.Join(testDirEvm, "eip-2537/add_G2_bls.json")
+	fail_add_G2_bls_tests			= filepath.Join(testDirEvm, "eip-2537/fail-add_G2_bls.json")
+
+	mul_G1_bls_tests				= filepath.Join(testDirEvm, "eip-2537/mul_G1_bls.json")
+	fail_mul_G1_bls_tests			= filepath.Join(testDirEvm, "eip-2537/fail-mul_G1_bls.json")
+	mul_G2_bls_tests				= filepath.Join(testDirEvm, "eip-2537/mul_G2_bls.json")
+	fail_mul_G2_bls_tests			= filepath.Join(testDirEvm, "eip-2537/fail-mul_G2_bls.json")
+
+	multiexp_G1_bls_tests			= filepath.Join(testDirEvm, "eip-2537/multiexp_G1_bls.json")
+	fail_multiexp_G1_bls_tests		= filepath.Join(testDirEvm, "eip-2537/fail-multiexp_G1_bls.json")
+	multiexp_G2_bls_tests			= filepath.Join(testDirEvm, "eip-2537/multiexp_G2_bls.json")
+	fail_multiexp_G2_bls_tests		= filepath.Join(testDirEvm, "eip-2537/fail-multiexp_G2_bls.json")
+
+	pairing_check_bls_tests			= filepath.Join(testDirEvm, "eip-2537/pairing_check_bls.json")
+	fail_pairing_check_bls_tests	= filepath.Join(testDirEvm, "eip-2537/fail-pairing_check_bls.json")
+
+	map_fp_to_G1_bls_tests			= filepath.Join(testDirEvm, "eip-2537/map_fp_to_G1_bls.json")
+	fail_map_fp_to_G1_bls_tests		= filepath.Join(testDirEvm, "eip-2537/fail-map_fp_to_G1_bls.json")
+	map_fp2_to_G2_bls_tests			= filepath.Join(testDirEvm, "eip-2537/map_fp2_to_G2_bls.json")
+	fail_map_fp2_to_G2_bls_tests	= filepath.Join(testDirEvm, "eip-2537/fail-map_fp2_to_G2_bls.json")
```

Comment:
Tabs are annoying. With 4 space width equivalent tabs the lines are aligned on the `=`...

Reply:
I agree, but gofmt dictatorship